src/FakeItEasy/Creation/IProxyOptions.cs

```diff
@@ -12,6 +12,6 @@ internal interface IProxyOptions
 
         IEnumerable<Action<object>> ProxyConfigurationActions { get; }
 
-        IEnumerable<CustomAttributeBuilder> AdditionalAttributes { get; }
+        IEnumerable<Expression<Func<Attribute>>> AdditionalAttributes { get; }

```

Comment:
If I understand `IProxyOptions` correctly then "additional" in `AdditionalInterfacesToImplement` makes sense because those interface are in addition to the originally specified interface, but with the attributes, they are not additional to anything, they are just the attributes.